docs/dqx/docs/installation.mdx

```diff
@@ -98,18 +98,29 @@ It is recommended to use serverless clusters for the workflows, as it allows for
 If serverless clusters are not used, a default cluster configuration will be used for the workflows.
 Alternatively, you can override the cluster configuration for each workflow in the `config.yml` file after the installation, to provide existing clusters to use.
 
-#### User vs Global Installation
+#### Installation Options
 
-DQX is installed by default in the user home directory (under `/Users/<user>/.dqx`). You can also install DQX globally
-by setting the 'DQX_FORCE_INSTALL' environment variable. The following options are available:
+DQX offers flexible installation options. By default, DQX is installed in the user home directory (under `/Users/<user>/.dqx`).
 
-* `DQX_FORCE_INSTALL=global databricks labs install dqx`: will force the installation to be for root only (`/Applications/dqx`)
-* `DQX_FORCE_INSTALL=user databricks labs install dqx`: will force the installation to be for user only (`/Users/<user>/.dqx`)
+**Custom Workspace Folder:**
+If you provide a custom path during installation (e.g., `/Shared/dqx-team` or `/Users/shared-user/dqx-project`), DQX will be installed there.
```

Comment:
The instruction should explain how to configure the custom path. I only shows the user and global installation options. 

It would also be worth to mention that group assigned clusters are not compatible with the user installation. For group assigned cluster, custom path needs to be used. Please test this manually as well on group assigned cluster.